tests/operator/operator.go

```diff
@@ -2952,6 +2952,8 @@ spec:
 			Expect(err).ToNot(HaveOccurred())
 			Eventually(ThisDeploymentWith(flags.KubeVirtInstallNamespace, components.VirtOperatorName), 180*time.Second, 1*time.Second).Should(HaveReadyReplicasNumerically("==", 0))
 
+			// Wait for at least one operator to be ready in order to observe an event
+			Eventually(ThisDeploymentWith(flags.KubeVirtInstallNamespace, components.VirtOperatorName), 180*time.Second, 1*time.Second).Should(HaveReadyReplicasNumerically(">", 0))
```

Comment:
Since this test will be removed with https://github.com/kubevirt/kubevirt/pull/10091/files#diff-1d8540ca83f201de58dfe667d5460041633dc6629709a3283e374af7d621976eL2953 anyway, I am not sure whether this change makes sense?

Reply:
You are on point. This is just an alternative if we would not remove the functionality for whatever reason.